### ISSUE TITLE
Analyze Project Portability menu option does not exist

### ISSUE THREAD

**Issue by TheBewilderedCoder** [dotnet/docs#23804]
The analyze project portability context menu and the Analyze Assembly Portability under Analyze menu do not exist as stated in the document.

[Edited to add doc details]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cdd8086b-9a95-38d5-160f-c12c653f954e
* Version Independent ID: 0f47d826-e3ef-8549-9d37-2e8afae29f46
* Content: [The .NET Portability Analyzer - .NET](https://docs.microsoft.com/en-us/dotnet/standard/analyzers/portability-analyzer#analyze-portability)
* Content Source: [docs/standard/analyzers/portability-analyzer.md](https://github.com/dotnet/docs/blob/main/docs/standard/analyzers/portability-analyzer.md)
* Product: **dotnet-fundamentals**
* Technology: **dotnet-analyzers**
* GitHub Login: @IEvangelist
* Microsoft Alias: **dapine**

**Comment by BillWagner** [comment]
@TheBewilderedCoder Thanks for raising this.

Can you add a link to the article this refers to? It looks like that block wasn't included in the issue. We're not sure which article should be updated.

**Comment by IEvangelist** [comment]
Hi 👋 @TheBewilderedCoder 
Thank you so much for posting this issue. We really appreciate it. I just tried re-creating this issue and I'm not able to.

Did you install the required Visual Studio extension?

https://marketplace.visualstudio.com/items?itemName=ConnieYau.NETPortabilityAnalyzer

Without this, there is no context menu. I just installed it, and I have it.

![image](https://user-images.githubusercontent.com/7679720/117019779-5c293b80-acbb-11eb-95a1-b1ab79c77558.png)
